ansible/roles/openshift_udp_limits/files/iptables_udp.conf

```diff
@@ -0,0 +1,5 @@
+# Ansible-managed
+:msg, startswith, "UDP:" -/var/log/iptables.log
```

Comment:
@portante are you concerned about other kernel messages starting with `UDP: `? I looked and our current kernel doesn't have any log messages that begin with `UDP: `. Even so, if we think it's a real concern, we could do `UDP outlog: ` or something like that. What do you think @rhdedgar?